### PR TITLE
refactor: remove run_worker_loop

### PR DESCRIPTION
**Pull request by bartlomieju**
Refactored to do the loop inside `WebWorker::poll`

### CODE REVIEW

File: cli/ops/worker_host.rs [hunk 1]
```diff
@@ -189,10 +127,11 @@ fn run_worker_thread(
     // TODO(bartlomieju): this thread should return result of event loop
     // that means that we should store JoinHandle to thread to ensure
     // that it actually terminates.
-    run_worker_loop(&mut rt, &mut worker).expect("Panic in event loop");
+    rt.block_on(worker).expect("Panic in event loop");
```
Comment (ry) [inline cli/ops/worker_host.rs]: use tokio_util::run_basic(worker)

File: cli/ops/worker_host.rs [hunk 2]
```diff
@@ -258,9 +202,10 @@ fn op_host_terminate_worker(
   let args: WorkerArgs = serde_json::from_value(args)?;
   let id = args.id as u32;
   let mut state = state.borrow_mut();
-  let worker_handle =
+  let (join_handle, worker_handle) =
     state.workers.remove(&id).expect("No worker handle found");
   worker_handle.terminate();
+  join_handle.join().expect("Panic in worker thread");
```
Comment (ry) [inline cli/ops/worker_host.rs]: Good

File: cli/state.rs [hunk 3]
```diff
@@ -267,7 +266,7 @@ impl State {
       metrics: Metrics::default(),
       global_timer: GlobalTimer::new(),
       workers: HashMap::new(),
-      next_worker_id: AtomicUsize::new(0),
+      next_worker_id: 0,
```
Comment (ry) [inline cli/state.rs]: 👍

File: cli/ops/worker_host.rs (hunk 1 above)
Comment (bartlomieju) [reply cli/ops/worker_host.rs]: Doesn't make sense here, need runtime to load modules beforehand

**Pull request merged**